DOCS-10828 teams (June 12) (#136)
mulesoft/docs-exchange: modules/ROOT/pages/to-set-permissions.adoc

endif::[]
:imagesdir: ../assets/images/

Exchange provides multiple ways to set up access management to provide collaboration across teams and meet governance requirements. Anypoint Platform comes with predefined permissions and roles that can be set in the Access Management section of Anypoint Platform to provide users with different access levels.

== Exchange Permissions

> To control access to assets in a business group, add users or teams to Exchange permissions role groups.

Reviewer: "permissions roles groups" sounds overloaded: permissions groups? role groups? roles?

Author: Changed to "role groups for Exchange permissions."

The following permissions pertain to every business group level:

Exchange Viewers:: Enables users to view and download assets within a business group. Users with this permission cannot add new assets, edit asset portal content, or share an asset with another user. Assign this permission to those who consume your assets in a specific business group.
Exchange Contributors:: Enables users to view, create, and download assets within a business group. Users with this permission can edit asset portal content in an existing asset version. Use this permission to invite users to edit and maintain your asset portal descriptions.
Exchange Administrators:: Enables users to view, create, and download assets within a business group. Users with this permission have all the same access as users with the Exchange contributor and Exchange viewer permissions, and access to share an asset with another user, deprecate an asset, and delete an asset. Use this permission to set up Exchange administrators for all the assets within a specific business group.

== Asset Permissions

> To control access to individual assets, add users or teams to asset permissions role groups.

Reviewer: even more overloaded: "asset permissions role groups"
See comment Line 11

Author: Fixed.

> The following permissions extend access on assets in addition to the permissions that a user has at the business group level:

Reviewer: ```suggestion
The following permissions extend access to assets in addition to the permissions that a user has at the business group level:
```

Author: Fixed.

Asset Viewers:: Enables users to view and download an asset. Users with this permission cannot edit asset portal content or share an asset with another user. Use this permission to invite a user outside your business group to view and download an asset.
Asset Contributors:: Enables users to view, add a new version, and download an asset. Use this permission to invite a user outside your business group to view, download, and add edit portal content for an asset.

> Asset Administrators:: Enables users to view, create, download, deprecate, and delete an asset. Users with this permission have the same access as users with the Exchange Administrators permission, but on only a single asset. This permission is assigned by default to an asset creator. Use this permission to extend administrator permissions on an asset to an additional user.

Reviewer: ```suggestion
Asset Administrators:: Enables users to view, create, download, deprecate, and delete an asset. Users with this permission have the same access as users with the Exchange Administrators permission, but on only a single asset. This permission is assigned by default to an asset creator. Use this permission to extend administrator permissions to an asset to an additional user.
```

Author: Fixed.

endif::[]
:imagesdir: ../assets/images/

> Exchange provides multiple ways to set up access management to provide collaboration across teams and meet governance requirements. Anypoint Platform comes with predefined permissions and role groups that can be set in Access Management to provide users with different access levels.

Reviewer 2: ```suggestion
Exchange provides multiple ways to set up access management to provide collaboration across teams and meet governance requirements. Anypoint Platform comes with predefined permissions and roles that can be set in the Access Management section of Anypoint Platform to provide users with different access levels.
```

Author: Fixed.

== Exchange Permissions

> Exchange permissions are the best way to control access to assets in a business group. Before Exchange permissions were introduced, very similar Exchange role groups controlled Exchange access, and role groups are still supported.

Reviewer 2: ```suggestion
Use Exchange permissions to control access to assets in a business group. You can assign Exchange permissions to individual users or by using the xref:access-management::teams.adoc[Teams] feature.
```

Reviewer 2: I *think* you can use teams for this? Please verify.

Author: Fixed.

> These permissions pertain to every Business Group level:

Reviewer 2: ```suggestion
The following permissions pertain to every business group level:
```

Author: Fixed.

> * Exchange Viewers: This permission enables users to view and download assets within a business group. The users with this permission cannot add new assets, edit asset portal content, or share an asset with another user. Assign this permission to those who consume your assets in a specific business group.

Reviewer 2: ```suggestion
Exchange Viewers:: Enables users to view and download assets within a business group. Users with this permission cannot add new assets, edit asset portal content, or share an asset with another user. Assign this permission to those who consume your assets in a specific business group.

```

Author: Fixed.

> * Exchange Contributors: This permission enables users to view, create, and download assets within a business group. Users with this permission can edit asset portal content in an existing asset version. Use this permission to invite users to edit and maintain your asset portal descriptions.

Reviewer 2: ```suggestion
Exchange Contributors:: Enables users to view, create, and download assets within a business group. Users with this permission can edit asset portal content in an existing asset version. Use this permission to invite users to edit and maintain your asset portal descriptions.

```

Author: Fixed.

> * Exchange Administrators: This permission enables users to view, create, and download assets within a business group. Users with this permission have all the same access as users with the Exchange contributor and Exchange viewer permissions, and access to share an asset with another user, deprecate an asset, and delete an asset. Use this permission to set up Exchange administrators for all the assets within a specific business group.

Reviewer 2: ```suggestion
Exchange Administrators:: Enables users to view, create, and download assets within a business group. Users with this permission have all the same access as users with the Exchange contributor and Exchange viewer permissions, and access to share an asset with another user, deprecate an asset, and delete an asset. Use this permission to set up Exchange administrators for all the assets within a specific business group.

```

Author: Fixed.

== Asset Permissions

> Asset permissions are the best way to control access to individual assets. Before asset permissions were introduced, very similar asset role groups controlled asset access, and role groups are still supported.

Reviewer 2: ```suggestion
Use asset permissions to control access to assets in a business group. You can assign asset permissions to individual users or by using the xref:access-management::teams.adoc[Teams] feature.
```

Reviewer 2: I *think* you can use teams for this? Please verify.

Author: Fixed.

> These permissions extend access on assets in addition to the permissions that a user has at the business group level:

Reviewer 2: ```suggestion
The following permissions extend access on assets in addition to the permissions that a user has at the business group level:
```

Author: Fixed.

> * Asset Viewers: This permission enables users to view and download an asset. Users with this permission cannot edit asset portal content or share an asset with another user. Use this permission to invite a user outside your business group to view and download an asset.

Reviewer 2: ```suggestion
Asset Viewers:: Enables users to view and download an asset. Users with this permission cannot edit asset portal content or share an asset with another user. Use this permission to invite a user outside your business group to view and download an asset.

```

Author: Fixed.

> * Asset Contributors: This permission enables users to view, add a new version, and download an asset. Use this permission to invite a user outside your business group to view, download, and add edit portal content for an asset.

Reviewer 2: ```suggestion
Asset Contributors:: Enables users to view, add a new version, and download an asset. Use this permission to invite a user outside your business group to view, download, and add edit portal content for an asset.

```

Author: Fixed.

> * Asset Administrators: This permission enables users to view, create, download, deprecate, and delete an asset. Users with this permission have the same access as users with the Exchange Administrators permission, but on only a single asset. This permission is assigned by default to an asset creator. Use this permission to extend administrator permissions on an asset to an additional user.

Reviewer 2: ```suggestion
Asset Administrators:: Enables users to view, create, download, deprecate, and delete an asset. Users with this permission have the same access as users with the Exchange Administrators permission, but on only a single asset. This permission is assigned by default to an asset creator. Use this permission to extend administrator permissions on an asset to an additional user.
```

Author: Fixed.
